controllers/podnetworkchaos/types_test.go

```diff
@@ -90,7 +90,11 @@ func TestHostNetworkOption(t *testing.T) {
 		}
 		objs = append(objs, chaos)
 
-		fakeClient := fake.NewClientBuilder().WithScheme(provider.NewScheme()).WithRuntimeObjects(objs...).Build()
+		fakeClient := fake.NewClientBuilder().
+			WithScheme(provider.NewScheme()).
+			WithRuntimeObjects(objs...).
+			WithStatusSubresource(pods...).
+			Build()
```

Comment:
`WithStatusSubresource` should be used with the type of the Resource which contains a "status subresource" as the parameter, to register / mark this type  including a subresource.

if should be used like `WithStatusSubresource(&v1alpha1.PodNetworkChaos{})`

It could resolve this testcase.